app/scripts/modules/netflix/report/reservationReport.component.spec.ts

```diff
@@ -1,26 +1,38 @@
-'use strict';
+import {ICompileService, IHttpBackendService, IQService, IRootScopeService, mock} from 'angular';
 
 import {matchers} from '../../../../../test/helpers/customMatchers';
 import {SETTINGS} from 'core/config/settings';
+import {RESERVATION_REPORT_COMPONENT} from './reservationReport.component';
+import {ReservationReportReader} from 'netflix/report/reservationReport.read.service';
+import {AccountService} from 'core/account/account.service';
 
 describe('Directives: reservation report', function () {
 
-  require('./reservationReport.directive.html');
-
-  beforeEach(window.module(require('./reservationReport.directive')));
+  require('./reservationReport.component.html');
+  beforeEach(mock.module(RESERVATION_REPORT_COMPONENT));
   beforeEach(function() {
     jasmine.addMatchers(matchers);
   });
 
-  beforeEach(window.inject(function ($rootScope, $compile, reservationReportReader, $httpBackend, accountService, $q) {
```

Comment:
I think this was an errant comment?  Only function I can see here is for the `addMatchers`.